values.yaml

```diff
@@ -634,6 +634,26 @@ keycloak:
   extraVolumeMounts: |
     - name: incore-theme
       mountPath: /opt/bitnami/keycloak/themes/incore-theme
+    
+
+# ----------------------------------------------------------------------
+# APPROVAL APP CONFIGURATION
+# ----------------------------------------------------------------------
+approval:
+  replicas: 1
+  image:
+    repository: approval-app
+    tag: latest
+  service:
+    type: ClusterIP
+    port: 5000
+  ingress:
+    annotations:
+  ADMIN_LIST: jonglee1@illinois.edu,cmnavarr@illinois.edu
+  KEYCLOAK_USERNAME: keycloak
+  EMAIL_FROM: no-reply@illinois.edu
+  SMTP_SERVER: outbound-relays.techservices.illinois.edu
```

Comment:
Maybe we can put the pseudo email and server in this and put the real values in values-incore-*.yaml?

Reply:
Same as above.